src/lib/pickles/plonk_checks/plonk_checks.ml

```diff
@@ -257,6 +261,7 @@ module Make (Shifted_value : Shifted_value.S) (Sc : Scalars.S) = struct
         ; generic
         }
 
+  (** Checks that the scalars were computed correctly (now that we can do operations in this field). These operations were deferred as the verifier circuit was previously operating in a different field. *)
```

Comment:
Suggestion:
```ocaml
(** Check that the computed proof scalars match the expected ones, using the
    native field.
    Note that the expected scalars are used to check the linearization in a
    proof over the other field, where those checks are more efficient, but we
    deferred the arithmetic checks until here so that we have the efficiency of
    the native field .
*)
```